diffpy/pdfmorph/tests/test_morph_func.py

```diff
@@ -3,6 +3,31 @@
 from diffpy.pdfmorph import pdfmorph, morph_default_config
 from diffpy.pdfmorph.tests.test_morphstretch import heaviside
 
+
+def test_morphfunc_verbose():
+    lb, ub = 1, 2
+    xref = np.arange(0.01, 5, 0.01)
+    yref = heaviside(xref, lb, ub)
+    # expand 30%
+    stretch = 0.3
+    xobj = xref.copy()
+    yobj = heaviside(xref, lb * (1 + stretch), ub * (1 + stretch))
+    cfg = morph_default_config(stretch=0.1) # off init
+    morph_rv = pdfmorph(xobj, yobj, xref, yref, verbose=True, **cfg)
```

Comment:
Is there some assertion that these tests verify?

Reply:
It is a smoke test. I was trying to test if the verbose option inside `pdfmorph` function works. Inspired by the f-string problem, I feel it might not be a bad idea to test against py2 and older python :p